run

```diff
@@ -68,6 +68,38 @@ if [ ! -z "$OPENDKIM_DOMAINS" ] ; then
   dkimConfig
   service opendkim start
 fi
+
+# Checking for user specified passwd file
+if [ ! -z "$SASL_Passwds" ] ; then
+  # Creating auth settings file
+  if [ ! -e /etc/postfix/sasl/smtpd.conf ] ; then
+    cat <<'EOF' > /etc/postfix/sasl/smtpd.conf
+pwcheck_method: saslauthd
+mech_list: CRAM-MD5 DIGEST-MD5 LOGIN PLAIN
+EOF
+  fi
+  
+  # Updating saslauthd config to work with chroot
+  sed -i 's|START=no|START=yes|' /etc/default/saslauthd
+  sed -i 's|OPTIONS="-c -m /var/run/saslauthd"|OPTIONS="-c -r -m /var/spool/postfix/var/run/saslauthd"|' /etc/default/saslauthd
+  dpkg-statoverride --add root sasl 710 /var/spool/postfix/var/run/saslauthd
+  
+  # Giving postfix permission to use saslauthd
+  adduser postfix sasl
```

Comment:
sed, dpkg and adduser should be fine running if the change has already been made?

Reply:
Yeah, no issues.

sed is looking for a string to replace so once the string is replaced on the first run it's not going to do anything on future runs.
dpkg and adduser recognize that they ran before and acknowledge it in the container logs:
```
dpkg-statoverride: error: an override for '/var/spool/postfix/var/run/saslauthd' already exists; aborting
The user `postfix' is already a member of `sasl'.
```